templates/function.tpl

```diff
@@ -2,4 +2,5 @@
  * {{name}}
  *{{#parameters}}
  * @param {{type}}{{^type}}mixed{{/type}} ${{name}}{{/parameters}}
- */
+ {{#return}}* @return {{return}} 
```

Comment:
Let me suggest to change this to
```
{{#return}}* @return ${{{?func: vmustache#IncrementCounter("params")}}:{{return}}} ${{{?func: vmustache#IncrementCounter("params")}}}
```
to define additional two tab-stops and have the behaviour identical with tab-stops on the `@param` lines.